os/kernel/binary_manager/binary_manager_data.c

```diff
@@ -191,6 +191,13 @@ void binary_manager_add_binlist(FAR struct tcb_s *tcb)
 {
 	int bin_idx;
 
+	DEBUGASSERT(tcb);
```

Comment:
For line 196, we'd better check the tcb not only when debug.

Reply:
Fixed.

Reply:
I think it should not go panic because it is not fault case. A tcb can be NULL in normal scenario as you said.

Reply:
True. I removed the assert now.